### ISSUE TITLE
[GH Request] Archive the openedx fork of django-ratelimit-backend

### ISSUE THREAD

**Issue by ormsbee** [openedx/tcril-engineering#253]
### Firm Name

tCRIL

### Urgency

Low

### Problem/Request

Per @jmbowman:

> There is a fork of django-ratelimit-backend in the openedx org.  We stopped using this package before the Nutmeg branches were cut.  Does someone at tCRIL want to archive it now, or set a reminder to do so once Nutmeg is released and Maple falls out of support?

-----

I'm going to announce it now in the forums and give it a 1.5 week notice until removal (only because everyone's at the conference, and there's a chance that folks won't see it until next week). This is low priority, so the timing is not urgent anyhow.

### Reasoning

This package is no longer used in the Nutmeg release.

**Comment by ormsbee** [comment]
Archiving target date: 2022-05-04

**Comment by ormsbee** [comment]
Announced in https://discuss.openedx.org/t/archiving-the-openedx-django-ratelimit-backend-repo/7102

**Comment by jmbowman** [comment]
@ormsbee With Nutmeg now released, is this just waiting on time to come back and archive it?

**Comment by Carlos-Muniz** [comment]
`django-ratelimit-backend` has been archived and org ownership has been transferred to `openedx-unsupported`.
